vue/components/ui/molecules/scrollable-items/scrollable-items.vue

```diff
@@ -0,0 +1,260 @@
+    scrollbar-width: none;
+    white-space: nowrap;
+}
+
+.scrollable-items > .items-container::-webkit-scrollbar {
+    display: none;
+}
+
+.scrollable-items > .items-container > .item {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 28px;
+    margin: 0px 12px 0px 0px;
+    opacity: 0.5;
+    transition: opacity 0.2s ease-out;
+}
+
+.scrollable-items > .items-container > .item.last {
+    margin: 0px 0px 0px 0px;
+}
+
+.scrollable-items > .items-container > .item:hover,
+.scrollable-items > .items-container > .item.hover {
+    opacity: 0.7;
+}
+
+.scrollable-items > .items-container > .item.selected {
+    opacity: 1;
+}
+</style>
+
+<script>
+export const ScrollableItems = {
+    name: "scrollable-items",
+    props: {
+        /**
+         * List of scrollable items. Example:
+         * [{ label: "Item 1", value: "item1" },
+         * { label: "Item 2", value: "item2" }].
+         */
+        items: {
+            type: Array,
+            default: () => []
+        },
+        /**
+         * Item currently selected. Example: "item1".
+         */
+        selected: {
+            type: String,
+            default: null
+        },
+        /**
+         * Enables/Disables the previous/next arrow buttons that are shown in
+         * the left and right side of component.
+         */
+        arrows: {
+            type: Boolean,
+            default: true
+        },
+        /**
+         * If set to true, the arrows will disappear when there is enough space
+         * for all the items to be visible.
+         */
+        autoHideArrows: {
+            type: Boolean,
+            default: true
+        },
+        /**
+         * Dictates how the items are horizontally aligned. Example: "center".
+         */
+        alignment: {
+            type: String,
+            default: "center"
+        },
+        /**
+         * Dictates the horizontal scroll speed of the items when using the mouse
+         * scroll wheel. Negative values inverts the default direction of the scroll.
+         */
+        scrollSpeed: {
+            type: Number,
+            default: 10
+        },
+        /**
+         * Props for the arrow button icon components.
+         */
+        buttonIconProps: {
+            type: Object,
+            default: () => ({})
+        }
+    },
+    data: function() {
+        return {
+            selectedData: this.selected,
+            isMouseDown: false,
+            isDraggingItems: false,
+            scrollableWidth: 0
+        };
+    },
+    computed: {
+        itemsContainerStyle() {
+            return {
+                "text-align": this.alignment
+            };
+        },
+        _buttonIconProps() {
+            return {
+                borderRadius: 0,
+                iconStrokeWidth: 3,
+                ...this.buttonIconProps
+            };
+        },
+        showArrows() {
+            const hideArrows = this.autoHideArrows && !(this.scrollableWidth > 0);

```

Comment:
Do it in the positive
```suggestion
            const hideArrows = this.autoHideArrows && this.scrollableWidth <= 0;
```